### PR TITLE
Add Area Line Chart Documentation

### PR DESCRIPTION
**Pull request by SyafiqMuizz**

### CODE REVIEW

**Comment by tchyong** [comment]
@SyafiqMuizz please ensure you 1 PR for 1 purpose. Please remove pie/donut chart related from this PR. 
Close this PR and resubmit new PR if needed.

File: visual/area-line-graph.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+|---|---|
+|X-Axis|Year|
+|Y-Axis|Population|
+|Group|Animals|
+
+Displayed within the chart are the total number of wildlife animals remaining, where each of the animals have a varied counts throughout the years, where Bears have an exponentially increasing count throughout the years, as opposed to Dolphins which are quickly declining in their population year after year. Not only that, the Whales have a constant increase and decrease in their count throughout the years of 2012 to 2017.
+
+**Output**
+
+![Wildlife Population](./images/area-line-graph/output-2.PNG)
```
Comment (tchyong) [inline visual/area-line-graph.md]: made decimal point to 0 as it is measurement for living thing and should be real number (no decimal)

File: visual/area-line-graph.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+|X-Axis|Year|
+|Y-Axis|Revenue(in Millions)|
+
+Displayed within the line chart, the line graph has a steady increase from its initial point to its end point. This can be said as Nike has a constant growth in their revenue as the years pass on. Their highest revenue count is from the year of 2019, where they earned 39.117 million US dollars on their revenue, which is an 8.3% increase in revenue as compared to 2018, where had 36.397 million US dollars in revenue. As the revenue is on a steady path of increase, Nike could continue their current marketing and business plans in the future to gain a steady revenue increase, along with setting the trend line as a guide to see which years had the most revenue increase when compared to their previous years, where they could implement their marketing strategies from said years into future endeavours.
+
+**Output**
+
+![Nike Revenue](./images/area-line-graph/output-1.PNG)
+
+### Wildlife Population throughout the Years (Area Chart)
```
Comment (tchyong) [inline visual/area-line-graph.md]: this data set is for specific area/country? 
As the wildlife count for dolphin is too less, only 1 at year 2017 seem does not make sense.

File: visual/area-line-graph.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+**Output**
+
+![Wildlife Population](./images/area-line-graph/output-2.PNG)
+
+**Analysis**
+
+The conclusion of the use case chart displays the population of three different animals within the years of 2012 to 2017, where the highest count of population is Bears in 2017, having the count of 184, where it has an increasing number from their initial starting point of 2012. Another note to point out is the count of Dolphins drastically depleting within the timeline of 2012 to 2017, having the count of only 1 in the year of 2017, whereas it had the highest count of 150 in their initial year when compared to the other animals. The Whales population have a steady population count year by year, ranging their population count from 54 - 100. By analysing this graph, wildlife conservation organisations could take the initiative to focus on preserving the Dolphins due to their very low count as of 2017, finding methods to save their environment to increase the count of Dolphins for the future years. Furthermore, the same could be said for the Whales count as the constant increase and decrease of their count could be improved to have a steady growth in their population.
```
Comment (tchyong) [inline visual/area-line-graph.md]: the protection step for wildlife success when the count growth per year. For those decrease mean the effort made previously is not success, should come out new effort and pay more attention to preserving the wildlife that going to distinct

File: visual/area-line-graph.md (hunk 2 above)
Comment (izairiabdlatif) [reply visual/area-line-graph.md]: @tchyong 

> this data set is for specific area/country?
> As the wildlife count for dolphin is too less, only 1 at year 2017 seem does not make sense.

Not going to make changes for this, as dolphin number =1 in 2017 shows that they were becoming extinct OR have migrated elsewhere from the venue or research

**Comment by izairiabdlatif** [comment]
@tchyong  Changes made to the document, ready for review

**Pull request merged**